test/Query.test.ts

```diff
@@ -0,0 +1,69 @@
+import { gql } from "graphql-tag";
+import { ApolloServer } from "apollo-server-express";
+import { resolvers } from "../src/resolvers";
+import fs from "fs";
+const { expect } = require("chai");
+
+const typeDefs = fs.readFileSync("./src/schema/schema.graphql", "utf8");
+
+it("Query Example Test", async () => {
+	const testServer = new ApolloServer({
+		typeDefs,
+		resolvers,
+	});
+
+	let result = await testServer.executeOperation({
+		query: gql`
+			query {
+				me {
+					id
+					username
+					password
+					role
+					preferences {
+						id {
+							subject
+							code
+							term
+						}
+						preference
+					}
+					active
+				}
+			}
+		`,
+	});
+
+	//Will pass now as 'me' is giving null for now
+	expect(result.data?.me).to.not.exist;
+	expect(result.data?.me).to.be.null;
+	// expect(result.data?.me.id).to.be.a("number");
```

Comment:
Try and use `describe` blocks and `it` blocks.
`describe('Query')` and `it('works which there is no user logged in')` etc.